pkg/skaffold/render/renderer/renderer.go

```diff
@@ -39,25 +39,15 @@ import (
 )
 
 const (
-	DefaultHydrationDir = ".kpt-pipeline"
-	dryFileName         = "manifests.yaml"
+	dryFileName = "manifests.yaml"
 )
 
 type Renderer interface {
-	Render(ctx context.Context, out io.Writer, artifacts []graph.Artifact, offline bool, outputPath string) error
+	Render(ctx context.Context, out io.Writer, artifacts []graph.Artifact, offline bool, noop string) error
```

Comment:
nit: maybe add a small comment saying this parameter is still necessary for the old-style `Render()` call

Reply:
Done. Also added the comment in the interface func `Runner.Render`